tracer/src/Datadog.Trace/Tagging/WebTags.cs

```diff
@@ -21,9 +21,6 @@ internal partial class WebTags : InstrumentationTags, IHasStatusCode
         [Tag(Trace.Tags.HttpRequestHeadersHost)]
         public string HttpRequestHeadersHost { get; set; }
 
-        [Tag(Trace.Tags.HttpUrl)]
```

Comment:
Hey, I'm not a fan of not declaring that there could be a HttpUrl in WebTags.
Can't we leave it?